test/jasmine/tests/gl2d_click_test.js

```diff
@@ -627,6 +627,9 @@ describe('hover with (x|y)period positioning', function() {
 
     it('@gl shows hover info for scattergl', function(done) {
         Plotly.newPlot(gd, require('@mocks/gl2d_period_positioning.json'))
+        .then(function() { return Plotly.restyle(gd, 'xperiodalignment', 'start'); })
+        .then(function() { return Plotly.restyle(gd, 'yperiodalignment', 'start'); })
```

Comment:
Nonblocking, but these tests can be simplified (and sped up) with the object form
```
Plotly.restyle(gd,{xperiodalignment: 'start', yperiodalignment: 'start'})
```

Reply:
Done in https://github.com/plotly/plotly.js/pull/5836/commits/17215ebd0d5d6cf13d0da1a5a5ce07b490741c7a